core/shared/src/main/scala/matryoshka/data/option.scala

```diff
@@ -25,6 +25,27 @@ trait OptionInstances {
   implicit def optionBirecursive[A]
       : Birecursive.Aux[Option[A], Const[Option[A], ?]] =
     id.idBirecursive[Option[A]]
+
+  implicit val optionDelayEqual: Delay[Equal, Option] =
+    new Delay[Equal, Option] {
+      def apply[A](a: Equal[A]) = {
+        implicit val aʹ: Equal[A] = a
+        Equal[Option[A]]
+      }
+    }
+
+  implicit val optionDelayOrder: Delay[Order, Option] =
```

Comment:
Apologies for not being more explicit, but my comment on `Equal[Option[A]]` also applies to `Order[Option[A]]` and `Show[Option[A]]`.